base/multimedia.jl

```diff
@@ -16,18 +16,48 @@ export AbstractDisplay, display, pushdisplay, popdisplay, displayable, redisplay
 """
     MIME
 
-A type representing a standard internet data format. "MIME" stands for
-"Multipurpose Internet Mail Extensions", since the standard was originally
-used to describe multimedia attachments to email messages.
+A parametric type representing a standard internet data format.
```

Comment:
```suggestion
A parametric type representing a content type/format, which is used to define [`show`](@ref)
methods to output objects in different formats and allows [`display(x)`](@ref) to detect
which formats are available for a given `x` in a particular display environment.
```

Reply:
The `MIME` type is **only** used for the `show` function? Should the docstring limit the type's usage in this way?

Reply:
In Base, it is also used for `repr` and `showable` and `display` methods, IIRC.  However, the most common usage is to define `show` methods, since the other functions get methods automatically from `show` in most cases.